mbuild/formats/lammpsdata.py

```diff
@@ -161,23 +171,33 @@ def write_lammpsdata(structure, filename):
 
         # Atom data
         data.write('\nAtoms\n\n')
+        if atom_style == 'atomic':
+            atom_line = '{index:d}\t{type_index:d}\t{x:.6f}\t{y:.6f}\t{z:.6f}\n'
+        elif atom_style == 'charge':
+            atom_line = '{index:d}\t{type_index:d}\t{charge:.6f}\t{x:.6f}\t{y:.6f}\t{z:.6f}\n'
+        elif atom_style == 'molecular':
+            atom_line = '{index:d}\t{zero:d}\t{type_index:d}\t{x:.6f}\t{y:.6f}\t{z:.6f}\n'
+        elif atom_style == 'full':
+            atom_line ='{index:d}\t{zero:d}\t{type_index:d}\t{charge:.6f}\t{x:.6f}\t{y:.6f}\t{z:.6f}\n'
+
         for i,coords in enumerate(xyz):
-            data.write('{:d}\t{:d}\t{:d}\t{:.6f}\t{:.6f}\t{:.6f}\t{:.6f}\n'.format(i+1,0,unique_types.index(types[i])+1,charges[i],*coords))
-
-        # Bond data
-        if bonds:
-            data.write('\nBonds\n\n')
-            for i,bond in enumerate(bonds):
-                data.write('{:d}\t{:d}\t{:d}\t{:d}\n'.format(i+1,bond_types[i],bond[0],bond[1]))
-
-        # Angle data
-        if angles:
-            data.write('\nAngles\n\n')
-            for i,angle in enumerate(angles):
-                data.write('{:d}\t{:d}\t{:d}\t{:d}\t{:d}\n'.format(i+1,angle_types[i],angle[0],angle[1],angle[2]))
-
-        # Dihedral data
-        if dihedrals:
-            data.write('\nDihedrals\n\n')
-            for i,dihedral in enumerate(dihedrals):
-                data.write('{:d}\t{:d}\t{:d}\t{:d}\t{:d}\t{:d}\n'.format(i+1,dihedral_types[i],dihedral[0],dihedral[1],dihedral[2],dihedral[3]))
+            data.write(atom_line.format(index=i+1,type_index=unique_types.index(types[i])+1,zero=0,charge=charges[i],x=coords[0],y=coords[1],z=coords[2]))
```

Comment:
nit: can you break this line up and add spaces between args so that it's easier to read?